layout/templates/flipClock.mustache

```diff
@@ -0,0 +1,39 @@
+<div class="flip-clock-wrapper totalTime">
+  <div class="clockContainer">
+	  <div class="clockLabel">
+		  <span>Years</span>
+		</div>
+		<div class="clockNumbersContainer">
+		  {{#flips}}
```

Comment:
Looking at this again, this will have to be something like flips1 or we would just end up repeating ourselves (maybe flipgroup1).